test/e2e/suites/managed/eks_test.go

```diff
@@ -36,12 +36,13 @@ import (
 // General EKS e2e test.
 var _ = ginkgo.Describe("[managed] [general] EKS cluster tests", func() {
 	var (
-		namespace        *corev1.Namespace
-		ctx              context.Context
-		specName         = "eks-nodes"
-		clusterName      string
-		cniAddonName     = "vpc-cni"
-		corednsAddonName = "coredns"
+		namespace            *corev1.Namespace
+		ctx                  context.Context
+		specName             = "cluster"
+		clusterName          string
+		cniAddonName         = "vpc-cni"
+		corednsAddonName     = "coredns"
+		eksKubernetesVersion = "v1.21.14"
```

Comment:
Can we use [KUBERNETES_VERSION](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/test/e2e/data/e2e_eks_conf.yaml#L177) from e2e_eks_conf.yaml instead of specifying one here?

MachinePool uses k8s version of v1.21.14 while AWSManagedControlPlane uses v1.22. See [artifacts](https://gcsweb.k8s.io/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-aws/3094/pull-cluster-api-provider-aws-e2e-eks/1557132529587195904/artifacts/clusters/bootstrap/resources/cluster-fjcwdc/). Unless there is a specific reason for it, I think it will be great to be consistent.

Reply:
This was added a long time ago. I think at that time using `KUBERNETES_VERSION` didn't work, but maybe things changed and let me try again.

Reply:
Sounds great. This is not a blocker but a suggestion. If it takes too much time, we can refactor as a follow up.

Reply:
I removed it and now uses `KUBERNETES_VERSION` instead, it's tested and works well.